draft-tulshibagwale-oauth-transaction-tokens-03.md

```diff
@@ -308,28 +308,50 @@ The JWT body MUST have the following claims:
 
 * An `iss` claim, whose value is a URN {{RFC8141}} that uniquely identifies the workload or the Txn-Token Service that created the Txn-Token.
 * An `iat` claim, whose value is the time at which the Txn-Token was created.
+* An `aud` claim, whose value is a URN {{RFC8141}} that uniquely identifies the audience of the Txn-Token. This MUST identify the trust domain in which the Txn-Token is used.
 * An `exp` claim, whose value is the time at which the Txn-Token expires.
 * A `txn` claim, whose value is the unique transaction identifier as defined in Section 2.2 of {{RFC8417}}. When used in the transaction token, it identifies the entire call chain.
 * A `sub_id` claim, whose value is the unique identifier of the user or workload on whose behalf the call chain is being executed. The format of this claim MAY be a Subject Identifier as specified in {{SubjectIdentifiers}}.
 * An `azc` claim, whose value is a JSON object that contains values that remain constant in the call chain.
```

Comment:
Originally, I thought of `azc` to be more transaction context focused. Since we've moved a lot of those details to the `orig` claim. It may make sense to move this to `azd` for "authorization details". For me this maps better to the data we have in the `azc` claim in the example. Thoughts?